### PR TITLE
MCS-417 Set snapToGrid to False

### PR DESCRIPTION
**Pull request by ThomasSchellenbergNextCentury**
Fixes the movement bug that Falk reported today on Slack.

We will also need the corresponding Unity fix:
https://github.com/NextCenturyCorporation/ai2thor/pull/92

Also added some more terminal output for human input mode.

### CODE REVIEW

**Review by deanwetherby — APPROVED** [review]

File: machine_common_sense/controller.py [hunk 1]
```diff
@@ -1055,6 +1073,7 @@ def wrap_step(self, **kwargs):
             logs=True,
             renderDepthImage=self.__depth_masks,
             renderObjectImage=self.__object_masks,
+            snapToGrid=False,
```
Comment (brianpippin) [inline machine_common_sense/controller.py:1076]: I'm pretty sure we don't ever want snapToGrid, but is there an instance where we do, where perhaps this should be a config?  Or is that just a future down the road thought that probably won't ever be needed?

**Review by brianpippin — APPROVED** [review]

File: machine_common_sense/controller.py (hunk 1 above)
Comment (ThomasSchellenbergNextCentury) [reply machine_common_sense/controller.py:1076]: Good question for the team.  Personally, I don't think so?

Comment (deanwetherby) [reply machine_common_sense/controller.py:1076]: I can't imagine needing snap-to-grid. If we ever do, we talk about exposing it via config then.

Comment (brianpippin) [reply machine_common_sense/controller.py:1076]: I didn't think we would, just wanted to bring it up to make sure.

**Review by rartiss55 — APPROVED** [review]

**Pull request merged**